version.h.in

```diff
@@ -22,6 +22,10 @@
 #ifndef h2o__version_h
 #define h2o__version_h
 
+#ifdef HAS_GITREV_H
```

Comment:
Please add `H2O_` prefix.

I would also appreciate it if you could consider using same naming conventions for the names of the file and variables related.
It seems that now the files (generate_gitrev.sh and gitrev.h use "gitrev"), while the variable defined in the file uses "GIT_REVISION". Can't they always be "gitrev" ("GITREV") or "git-revision" ("GIT_REVISION")?